app/views/articles/index.html.erb

```diff
@@ -0,0 +1,16 @@
+
+<% @articles.each do |article| %>
+  <div id=header>
+    <p><%= article.header %></p>
+  </div>
+
+  <div id=subheader>
+    <p><%= article.subheader %></p>
+  </div>
+
+  <div id=body>
+    <p><%= article.body %></p>
+    <%=image_tag("trump.jpg")%>
```

Comment:
It might be better to have this image tag in its own div with something like `id=imageplaceholder`(but still part of the articles.each div). Doesn't necessarily matter, but probably cleaner